tests/analysis_tests/tests/src/CompletionTaggedTemplate.res

```diff
@@ -0,0 +1,8 @@
+@module("meh") @taggedTemplate
+external meh: (array<string>, array<string>) => string = "default"
```

Comment:
Could you make this return something else than a string? Like a custom module in the same file, with a few t-first functions only? Or just something that has less functions in it. 

Reason is the emitted test file gets filled with tons of functions to manipulate strings, which makes diffs harder later on. We do need the empty dot completion test here, but it'd be good to reduce the items output by it.